### ISSUE TITLE
Strange german translation

### ISSUE THREAD

**Issue by DSiekmeier** [racket/drracket#96]
Hi there,
in DrRacket v6.8, german and ealier there is a strange sentence in the translation i guess.
If you start the GUI and click on "Start" on the top, the message below is shown:
"Alle Ausdrücke are sind abgedeckt" which seems to be mixed german / english.

Kind Regards.

![grafik](https://cloud.githubusercontent.com/assets/17233627/23702035/d7cd2c7a-03f9-11e7-9bd6-34d09eb4dde9.png)

**Comment by rfindler** [comment]
@mikesperber , just in case you didn't see this one.

**Comment by stamourv** [comment]
Also, another string constant a few lines above that one seems to have a duplicated "nicht":
"Ja, und nicht nicht wieder fragen"

Giant caveat: I know very, very close to nothing about German.

**Comment by DSiekmeier** [comment]
There are some more things in the GUI which I would suggest to change:
- In menu "Datei": "Definition drucken..." should be "Definition**en** drucken..." because you also use the plural in for example "Definitionen speichern"
- In menu "Hilfe": "Über DrRacket" should be "Über DrRacket**...**" because it opens a new Window. 
- same with "Hilfezentrum"

I would do a Pull request but actually I don't know where the language files are located. ;-)
Kind Regards.

**Comment by rfindler** [comment]
I cannot speak to the German, but ellipses don't mean "opens a new window",
they mean "in order to do this action I will have to ask you for some more
information". So, e.g., the "File|New" menu item does not get ellipses.

**Comment by mfelleisen** [comment]
@rfindler The suggestion concerning plural is acceptable but the original text is fine too because there might be only one. For uniformity we should probably use “Definition(en) drucken/speichern”. [Man I would not have come up with these words.]

**Comment by DSiekmeier** [comment]
@rfindler Yeah, your are perfectly right! I did not know that detail.. Sorry for that.

For everyone who is interested: [The Microsoft UX Guidlines](https://msdn.microsoft.com/en-us/library/dn742392.aspx#usingellipses)

**Comment by rfindler** [comment]
No problem! Thanks for helping out with the translation.

Robby

**Comment by gus-massa** [comment]
It looks like this is already fixed. Can someone confirm and close it?
